iree/vm/list.c

```diff
@@ -686,3 +686,124 @@ iree_status_t iree_vm_list_register_types(void) {
   iree_vm_list_descriptor.type_name = iree_make_cstring_view("vm.list");
   return iree_vm_ref_register_type(&iree_vm_list_descriptor);
 }
+
+#define std_swap(a, b) \
```

Comment:
generally sketchy to put things in other namespaces (well, C-style namespaces :) so we'd want to prefix it as `iree_swap` so there's no chance of conflicts with any other dependency/system library.